propertyestimator/utils/serialization.py

```diff
@@ -330,3 +331,40 @@ def deserialize_quantity(serialized):
             quantity_unit *= (getattr(unit, unit_name) ** power)
     quantity = unit.Quantity(serialized['unitless_value'], quantity_unit)
     return quantity
+
+
+def serialize_force_field(force_field):
+
```

Comment:
Could you add docstrings and comments? I'm not super up to date on the old FF implementation, so it took me a while to figure out what this is doing and how.

Reply:
Must've missed these - thanks for pointing them out!